mailpoet/assets/js/src/newsletter_editor/behaviors/TextEditorBehavior.ts

```diff
@@ -9,12 +9,12 @@ import { BehaviorsLookup } from 'newsletter_editor/behaviors/BehaviorsLookup';
 import { App } from 'newsletter_editor/App';
 import { tinyMceAdd } from 'newsletter_editor/tinymce/mailpoet_shortcodes.js';
 
+
```

Comment:
I guess this extra newline was added by accident?

Reply:
I'm a bit confused with this comment, since no newline has been added to `line 12` as I see it. https://d.pr/i/J5WoIh

Reply:
Ah, I see what happened here. I reviewed this PR commit by commit, and I made this comment when an extra newline was added in https://github.com/mailpoet/mailpoet/pull/4610/commits/3a6740342becab43099dde22960a1219e55db5a9#diff-ef944cf4cc1566facf2c6d659dacb51dd061c3630a8c9ca7c9c40ccbabd590a5R12. But I missed that this newline was removed in the next commit: https://github.com/mailpoet/mailpoet/pull/4610/commits/7bec395200d19dae7d32853a4a4131050883360c#diff-ef944cf4cc1566facf2c6d659dacb51dd061c3630a8c9ca7c9c40ccbabd590a5L12